README.md

```diff
@@ -1,7 +1,7 @@
 # leaderboard
 
 > "leaderboard" is a simple web application that displays a table of names and scores and allows you to add and remove names and scores from that table.
-> ![leaderboard](https://user-images.githubusercontent.com/87186552/151043785-59ac7cb6-5545-4120-9b6f-1183399d1a50.png)
+> ![leaderboardlatest](https://user-images.githubusercontent.com/87186552/151335363-85db4e6d-d5bf-4449-aae8-d132792f1adc.png)
```

Comment:
- [ ] in the License part of the readme file you added the MIT license which is great, but the link to the actual license is not working, please have a look at [the Microverse template](https://github.com/microverseinc/readme-template) to see what it should look like.

Reply:
Thank you for your review, the changes have been applied.